DESCRIPTION

```diff
@@ -1,17 +1,17 @@
 Package: algorithmia
 Type: Package
 Title: Allows you to Easily Interact with the Algorithmia Platform
-Version: 0.2.0
-Date: 2019-07-31
-Author: James Sutton, James Athappilly
-Maintainer: James Athappilly <jamesa@algorithmia.com>
+Version: 0.3.0
+Date: 2020-09-28
+Author: James Sutton, James Athappilly, Taylor Raack
+Maintainer: Robert Fulton <rfulton@algorithmia.com>
```

Comment:
Robert Fulton is a special user account set up to forward email; talk to Taylor about this with questions.

Reply:
It's hilarious that we have to do this.. CRAN is weird haha